lib/galaxy/web/framework/middleware/error.py

```diff
@@ -208,6 +215,8 @@ def exception_handler(self, exc_info, environ):
             simple_html_error=simple_html_error,
             environ=environ,
         )
+        rval = cast(str, rval)  # we know handle_exception returns string because html=True
```

Comment:
This seems to be the only place where `handle_exception()` is called in our entire codebase, so maybe just drop the `html` parameter?